mod-orders-storage/examples/piece_collection.sample

```diff
@@ -18,6 +18,8 @@
         "receiptDate": "2020-01-10T00:00:00.000Z",
         "receivingStatus": "Unreceivable",
         "claimingInterval": 0,
+        "internalNote": "Should be edited bit later",
```

Comment:
this message tells us that we should do something later, so lets change to "Intrernal note for Send Claim action" and "External note for Send Claim action to share with Vendor"

Reply:
or you can remain this sample as is, and move this info to description